From 3801620674bb052fcf7acf03915999fd6f48dd66 Mon Sep 17 00:00:00 2001
From: Kamal Heib <kheib@redhat.com>
Date: Mon, 29 Apr 2024 18:54:39 -0400
Subject: [PATCH] Fix build failure

../../src/vma/util/vma_list.h:210:31: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
  210 |         vma_list_t<T, offset> (const vma_list_t<T, offset>& other) {
      |                               ^
../../src/vma/util/vma_list.h:210:31: note: remove the '< >'

Signed-off-by: Kamal Heib <kheib@redhat.com>
---
 src/vma/util/vma_list.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vma/util/vma_list.h b/src/vma/util/vma_list.h
index 0ce65e8e5268..32fdedbb826f 100644
--- a/src/vma/util/vma_list.h
+++ b/src/vma/util/vma_list.h
@@ -207,7 +207,7 @@ public:
 		}
 	}
 
-	vma_list_t<T, offset> (const vma_list_t<T, offset>& other) {
+	vma_list_t(const vma_list_t<T, offset>& other) {
 		if (!other.empty())
 			vlist_logwarn("Copy constructor is not supported for non-empty list! other.size=%zu", other.m_size);
 		init_list();
-- 
2.44.0

